presto-bigquery/pom.xml

```diff
@@ -73,6 +73,24 @@
                 <artifactId>commons-codec</artifactId>
                 <version>1.13</version>
             </dependency>
+
+            <dependency>
+                <groupId>io.grpc</groupId>
+                <artifactId>grpc-context</artifactId>
+                <version>1.64.0</version>
+            </dependency>
+
+            <dependency>
+                <groupId>io.grpc</groupId>
+                <artifactId>grpc-protobuf-lite</artifactId>
+                <version>1.64.0</version>
+            </dependency>
+
+            <dependency>
```

Comment:
Thanks for explaining @ZacBlanco . Would it be possible to add a simple comment here? Something like "required by xxx"?

Reply:
added